{"uuid": "7baf47b2-e336-4a7d-825f-0790dccb693f", "children": ["9b51a9cf-0f5f-4bfa-9583-c7932a22597f"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for firefox; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n\n", "trace": "  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"X:\\Automation Testing\\06. CredKart_Pytest_Revision\\testCases\\conftest.py\", line 29, in setup\n    driver = webdriver.Firefox()\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 58, in __init__\n    if finder.get_browser_path():\n       ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py\", line 47, in get_browser_path\n    return self._binary_paths()[\"browser_path\"]\n           ~~~~~~~~~~~~~~~~~~^^\n  File \"X:\\Automation Testing\\03. Pytest Framework\\.venv1\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py\", line 78, in _binary_paths\n    raise NoSuchDriverException(msg) from err\n"}, "start": 1754977156722, "stop": 1754977158347}], "start": 1754977156722, "stop": 1754977158422}